src/pages/_app.tsx

```diff
@@ -65,13 +66,21 @@ function MyApp({ Component, pageProps }: AppProps) {
   }, []);
 
   return (
-    <NostrContext.Provider value={{ ndk }}>
-      <ChakraProvider theme={theme} colorModeManager={colorModeManager}>
-        <QueryClientProvider client={queryClient}>
-          <Component {...pageProps} />
-        </QueryClientProvider>
-      </ChakraProvider>
-    </NostrContext.Provider>
+    <>
+      <Head>
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
```

Comment:
`user-scalable=0` is not considered good practice. Is it needed?

Reply:
Just to prevent unwanted zooming. we can remove it.

Reply:
Perhaps better to remove, don't want to remove ability to zoom in.

Reply:
Alright, I'll revert it.